Tensile/KernelWriter.py

```diff
@@ -104,8 +104,33 @@ def makeSchedule(self, kernel, tensorParametersA, tensorParametersB, localWriteE
     lastLoadIter = 0
     if kernel["EnableMatrixInstruction"] and kernel["ScheduleIterAlg"] == 3:
       numMfmaPerIter = kernel["MIWaveTile"][0] * kernel["MIWaveTile"][1] * kernel["InnerUnroll"]
+      # Can locally overrid these ######
+      # number of mfma between last localWrite and barrier
+      numMfmaBetweenLWandBarrier = 1 if kernel["MatrixInstM"] == 32 else 2
+      # number of global read instructions between 2 mfma
+      self.numGlobalReadInsPerMfma = 2 if kernel["MatrixInstM"] == 32 and not kernel["ProblemType"]["TLUA"] and not kernel["ProblemType"]["TLUB"] and kernel["TransposeLDS"] and not kernel["1LDSBuffer"] else 1
+      # TODO: use a function to return MatrixInstructionLatency
+      MatrixInstructionLatency = kernel["MatrixInstM"] // 2 - 2
+      LocalWriteLatency = tensorParametersA["localWriteInstruction"].IssueLatency*2
+      numReadPerVectorA = tensorParametersA["bpe"] * kernel["LocalReadVectorWidth"] // int(tensorParametersA["localReadInstruction"].blockWidth * 4)
+      numReadPerVectorB = tensorParametersB["bpe"] * kernel["LocalReadVectorWidth"] // int(tensorParametersB["localReadInstruction"].blockWidth * 4)
+      numA = kernel["InnerUnroll"]*(kernel["MIWaveTile"][0] * numReadPerVectorA) // tensorParametersA["localReadInstruction"].numOffsets
+      numB = kernel["InnerUnroll"]*(kernel["MIWaveTile"][1] * numReadPerVectorB) // tensorParametersB["localReadInstruction"].numOffsets
+      readsPerIter = numA + numB if self.numReadsIterCoalesced == 1 else 0
```

Comment:
From PR #1025:
readsPerIter = numA + numB

Reply:
the modification here is to enable LRVW.